packages/picasso/src/RichTextEditor/RichTextEditor.tsx

```diff
@@ -98,6 +112,13 @@ export const RichTextEditor = forwardRef<HTMLDivElement, Props>(
       dispatch
     })
 
+    const { counterMessage, error, handleCounterMessage } = useCounter({
```

Comment:
```suggestion
    const { counterMessage, counterError, handleCounterMessage } = useCounter({
```

I think in another PR we will have to add also general `error` for RichTextEditor itself

Reply:
Makes sense